plugins/kubernetes_ng/app/javascript/widgets/app/types/cluster.ts

```diff
@@ -70,6 +70,8 @@ export const ClusterSchema = z.object({
     os: z.boolean(),
     kubernetes: z.boolean(),
   }),
+
+  raw: z.any(), // Raw data for JSON viewer
```

Comment:
If the intention is to accept an object of any shape then I think we can use `z.object({}).passthrough();` trick because I think `z.any` would allow anything like `boolean`, `string` etc.